infrastructure/cdn-in-a-box/readiness/run.sh

```diff
@@ -32,7 +32,7 @@ done
 
 while true; do
     sleep 3
-    exampleURLs=($(to-get /api/${TO_API_VERSION}/deliveryservices | jq -r '.response[].exampleURLs[]'))
+    exampleURLs=($(to-get /api/${TO_API_VERSION}/deliveryservices | jq -r '.response[].exampleURLs[]' || true))
```

Comment:
To avoid the `|| true`, this can be
```bash
if ! exampleURLs=($(to-get "/api/${TO_API_VERSION}/deliveryservices" | jq -r '.response[].exampleURLs[]')) || [[ "${#exampleURLs[@]}" -eq 0 ]]; then
```